.gitignore

```diff
@@ -6,3 +6,4 @@ dist/*.js
 dist/*.css
 dist/img/*.svg
 data/data.js
+js/lib/id/*.js
```

Comment:
No problem!  I want them ignored too, but I realized yesterday that `index.html` uses `js/lib/id/index.js`.